package.json

```diff
@@ -19,6 +21,7 @@
     "@typescript-eslint/eslint-plugin": "2.25.0",
     "@typescript-eslint/parser": "2.25.0",
     "babel-plugin-styled-components": "1.10.7",
+    "babel-polyfill": "^6.26.0",
```

Comment:
I think you don't need this anymore but we probably should import `regenerator-runtime` explicitly (i also had to do this in my open PR so it'll get dealt with one way or another if you want to just merge)

Reply:
In the interest of moving forward, I'm going to merge as is but I'm happy to circle back on this if it doesn't come naturally from your open PR!